EMS/core-bundle/src/Controller/ContentManagement/EnvironmentController.php

```diff
@@ -432,6 +435,8 @@ public function addAction(Request $request): Response
                 } else {
                     $environment->setAlias($this->instanceId.$environment->getName());
                     $environment->setManaged(true);
+                    $length = $this->environmentRepository->counter();
+                    $environment->setOrderKey($length++);
```

Comment:
Is there no case where an environment has already an orderKey? Maybe surrounding these lines by a if?

Reply:
In this case we create an environment and it has no orderkey

Reply:
In this case we are creating an environment so it has no orderkey, but will put an if to verify if an existing environment has the orderkey we want to set